### ISSUE TITLE
Function name `scatter` conflics

### ISSUE THREAD

**Issue by mkborregaard** [JuliaParallel/DistributedArrays.jl#185]
The function name `scatter` in DistributedArrays conflicts with the `scatter` function in, say, 80% of all julia's plotting packages (for making scatter plots). Would it be possible to rename to something unambiguous, such as `distribute`?

**Comment by andreasnoack** [comment]
It's a pretty standard parallel computing concept/function though. I think a solution could be not to export the `SPMD` functions from the `DistributedArrays` module. Then this will only become an issue when people use `DistributedArrays.SPMD`.

**Comment by vchuravy** [comment]
Yes I think the SPMD code might actually be a good example for a separate package, (but still in the same repository.)
